src/core/layout/qgsabstractreportsection.cpp

```diff
@@ -170,6 +171,48 @@ void QgsAbstractReportSection::reloadSettings()
     mFooter->reloadSettings();
 }
 
+bool QgsAbstractReportSection::accept( QgsStyleEntityVisitorInterface *visitor ) const
+{
+  if ( mParent && !visitor->visitEnter( QgsStyleEntityVisitorInterface::Node( QgsStyleEntityVisitorInterface::NodeType::ReportSection, QStringLiteral( "reportsection" ), QObject::tr( "Report Section" ) ) ) )
+    return true;
```

Comment:
I'll let you figure out the other visitEnter calls if this really should return false ;P

Reply:
Nope, it's correct. If a visitor returns false to visitEnter it means they don't want to visit that node, not that they want to abort the visitations (as per the dox)

Reply:
Got it